__init__.py

```diff
@@ -443,6 +444,16 @@ def on_gui_page_show(self, message):
         if 'mark-2' not in message.data.get('__from', ''):
             # Some skill other than the handler is showing a page
             self.has_show_page = True
+            
+            # If a skill overrides the animations do not show any
+            override_animations = message.data.get('__animations')
```

Comment:
It's probaly clearer if this is defaulted to False
`override_animations = message.data.get('__animations', False)`

or is there special significace if it's missing entirely?